electron/electron/handlers/runWhisper/runWhisper.ts

```diff
@@ -92,6 +95,11 @@ export default ipcMain.handle(
 
     console.log('RunWhisper: Running model with args', inputArray);
 
+    // Create a new env that has PythonUnbuffered set to true
+    // This will allow the child process to output to the console
+    const env = Object.create(process.env);
```

Comment:
As I understand this is not just creating a new env, but actively copying env variables from the electron process to the Python process, right? That's fine with me, just checking my understanding

Reply:
I think this might actually be redundant as it could just be an empty object with the PYTHONUNBUFFERED as its only K:V pair, but for now it passes dev env etc

Reply:
To me using a single entry object seems like a better separation of concerns, but I don’t feel strongly about it!

Reply:
Just checked it, looks like we need to pass a constructed env object that isnt just the pair. Not worth digging into now